README.md

```diff
@@ -154,6 +154,7 @@ require("diffview").setup({
   enhanced_diff_hl = false, -- See ':h diffview-config-enhanced_diff_hl'
   git_cmd = { "git" },      -- The git executable followed by default args.
   use_icons = true,         -- Requires nvim-web-devicons
+  auto_refresh = true,      -- Auto refresh the status of the current file
```

Comment:
Let's call it `watch_index`. And I think we should make the description a little bit more accurate:

```suggestion
  watch_index = true,       -- Update views and index buffers when the git index changes.
```